tests/runner.py

```diff
@@ -1354,38 +1350,49 @@ class BrowserCore(RunnerCore):
   def __init__(self, *args, **kwargs):
     super(BrowserCore, self).__init__(*args, **kwargs)
 
+  @staticmethod
+  def browser_open(url):
```

Comment:
looks like this is the other PR's change? would be good to keep them in separate PRs.

Reply:
oh, I missed the comment saying this can merge after that one, which resolves that issue. sounds good as is then.